paper/utils.md

```diff
@@ -0,0 +1,64 @@
+``` r
+library(funkyheatmap)
+library(tidyverse)
+```
+
+    ## ── Attaching core tidyverse packages ──────────────────────── tidyverse 2.0.0 ──
+    ## ✔ dplyr     1.1.4     ✔ readr     2.1.5
+    ## ✔ forcats   1.0.0     ✔ stringr   1.5.1
+    ## ✔ ggplot2   3.5.1     ✔ tibble    3.2.1
+    ## ✔ lubridate 1.9.4     ✔ tidyr     1.3.1
+    ## ✔ purrr     1.0.2     
+    ## ── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
+    ## ✖ dplyr::filter() masks stats::filter()
+    ## ✖ dplyr::lag()    masks stats::lag()
+    ## ℹ Use the conflicted package (<http://conflicted.r-lib.org/>) to force all conflicts to become errors
+
+``` r
+legends <- list(
+  list(title = "Bar", palette = "qc", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "benchmark", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "scaling", enabled = FALSE, geom = "bar"),
+  list(title = "Bar", palette = "stability", enabled = FALSE, geom = "bar"),
```

Comment:
I don't see these geoms in the svg; do we need to rerender?

Reply:
Nope, as you can see they're explicitly not enabled. If this is not added, `verify_legends` detects that there are palettes without legends and will plot these bar geoms.

Reply:
I see, good point. Would we want to add a legend for the different parts? The geoms were not added before mainly due to them not being available yet.

Reply:
I explicitly didn't want to, to recreate the relevant parts of this figure: https://www.nature.com/articles/s41587-019-0071-9/figures/8